src/main/kotlin/racingcar/Car.kt

```diff
@@ -1,8 +1,14 @@
 package racingcar
 
-data class Car(val position: Int = DEFAULT_POSITION) {
+data class Car(val name: String, val position: Int = DEFAULT_POSITION) {
 
-    fun move(isMove: Boolean): Car = if (isMove) Car(position + PLUS_POSITION) else this
+    fun move(isMove: Boolean): Car = if (isMove) {
+        this.copy(position = position + PLUS_POSITION)
+    } else {
+        this
+    }
```

Comment:
```suggestion
    fun move(isMove: Boolean): Car = if (isMove) this.copy(position = position + PLUS_POSITION) else this
```

어제 if 절에 대한 코딩 컨벤션을 찾아봤는데요. 블록으로 감싸주지 않는 것도 허용을 하더군요! 이건 스타일의 차이인 것 같으니, 선택을 하시면 될 것 같습니다!

유식님은 어떤 스타일을 선호하시나요? 저는 블록으로 감싸주는 스타일을 선호하지만, 요번 같은 로직에 대해서는 블록으로 감싸주니, 코드가 지저분해 보이네요.

그렇다고, when 절로 true false로 변경하면 좋지 않을까? 라는 생각을 해봤는데, 코틀린 코딩 컨벤션에서 when 절은 3개 이상인 경우에만 쓰라고 되어있더군요!

참고 
- [if else versus when](https://kotlinlang.org/docs/coding-conventions.html#if-versus-when)